packages/fxa-content-server/app/scripts/lib/channels/senders/web-channel.js

```diff
@@ -122,8 +122,9 @@ function formatEventDetail(win, eventDetail) {
   // See https://bugzilla.mozilla.org/show_bug.cgi?id=1275616 and
   // https://bugzilla.mozilla.org/show_bug.cgi?id=1238128
   if (
-    (userAgent.isFirefoxDesktop() || userAgent.isFirefoxAndroid()) &&
-    userAgent.browser.version >= 50
```

Comment:
@darktrojan the test failures were caused by the removal of the browser version condition.  I've restored it to the predicate, as my attempt at updating the related code became more time consuming than I expected, and this is a quicker fix.

Reply:
Really? Everything passed when I rebased it. That's weird. Oh well, as long as it works, and it appears to work.

Thanks!